elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/metric/AggregatedMetric.java

```diff
@@ -49,7 +49,18 @@ public AggregatedMetric(
             Class<?> fieldType,
             List<Class<? extends Aggregation>> aggregations
     ) {
-        super(schema, metricField, annotation, fieldType);
+        this(schema, metricField, annotation, fieldType, aggregations, metricField);
+    }
+
+    public AggregatedMetric(
+            Schema schema,
+            String metricField,
+            Meta annotation,
+            Class<?> fieldType,
+            List<Class<? extends Aggregation>> aggregations,
+            String columnName
```

Comment:
I don't think we should add columnName as a concept at this level.  If we need it, it should go into the SQL subclasses/extensions of these classes.